src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final ColorSupplier colorSupplier = new ColorSupplier();
+    private static final Random random = new Random();
+    private static final int ballCount = 100;
```

Comment:
Don't forget about naming rules for constants
```suggestion
    private static final int BALL_COUNT = 100;
```

Reply:
ok